tests/cache/tests.py

```diff
@@ -1117,6 +1118,30 @@ def test_delete_many_num_queries(self):
         with self.assertNumQueries(1):
             cache.delete_many(['a', 'b', 'c'])
 
+    def test_cull_num_queries(self):
+        """
+        There should be only one COUNT query
+        """
+        try:
+            cull_cache = caches['cull']
+        except InvalidCacheBackendError:
+            self.skipTest("Culling isn't implemented.")
+
```

Comment:
Is this really required?

Reply:
I'm not sure if this was required. I was following the pattern set by other tests in this file, but I didn't understand what the try/finally was actually doing. Happy to remove it, but feels safer to leave it.

Reply:
This is a "database cache"-specific tests, so it's not required. I removed it.